ddtrace/tracer/option.go

```diff
@@ -161,3 +161,12 @@ func WithError(err error) FinishOption {
 		cfg.Error = err
 	}
 }
+
+// NoDebugStack disable backtrace generation in case of error.
```

Comment:
Can you please use the wording suggested [here](https://github.com/DataDog/dd-trace-go/pull/355#discussion_r230328692)?
```go
// NoDebugStack prevents any error presented using the WithError finishing option
// from generating a stack trace. This is useful in situations where errors are frequent
// and performance is critical.
```